src/scenes/NftDetailPage/NftDetailModel.tsx

```diff
@@ -0,0 +1,44 @@
+import { useSetContentIsLoaded } from 'contexts/shimmer/ShimmerContext';
+import styled from 'styled-components';
+import { Nft } from 'types/Nft';
+import '@google/model-viewer'
+import './model-viewer.css';
+
+type Props = {
+  nft: Nft;
+};
+
+// TODO: Clean this up once fixed
+// https://github.com/google/model-viewer/issues/1502
+
+declare global {
+  namespace JSX {
+    interface IntrinsicElements {
+    'model-viewer': ModelViewerJSX &
+        React.DetailedHTMLProps<React.HTMLAttributes<HTMLElement>, HTMLElement>
+    }
+  }
+}
```

Comment:
i believe we want to put this in a `global.d.ts` file

Reply:
https://basarat.gitbook.io/typescript/project/modules/globals

Reply:
hmm or maybe not: https://stackoverflow.com/a/42257742/5377437

Reply:
resolved on discord